AuRoRA/src/scs/scs/cnc.py

```
@@ -38,303 +23,139 @@
-        Schedules async processing on the asyncio loop — never blocks.
-        """
         user_input = msg.data.strip()
-        if not user_input:
-            return
-
-        if self._busy:
-            self.get_logger().warning("⚠️  CNC busy — dropping input")
-            self._publish({"type": "error", "content": "GRACE is still thinking…"})
-            return
-
+        if not user_input or self._busy: return
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)